docs/moisture_dataset.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+
+import xarray
+
+def attribute_fix(ds):
+    """Needed to fix a low-level bug in ecCodes.
+    
+    Sometimes, shortNames get overloaded in ecCodes's table. 
+    To eliminate ambiguity in their string matching, we
+    force ecCodes to make use of the paramId, which is a
+    consistent source-of-truth.
+    """
+    for var in ds:
+        attrs = ds[var].attrs
+        result = attrs.pop('GRIB_cfName', None)
+        result = attrs.pop('GRIB_cfVarName', None)
+        result = attrs.pop('GRIB_shortName', None)
+        ds[var].attrs.update(attrs)
+    return ds
+
+
+def compute_gg_ll_dataset(ml_surface, ml_wind, ml_moisture, datestring):
+   """
+   This function takes a datestamp and the three AR model-level ERA5 
+   datasets to produce a unified model-level dataset on a Gaussian Grid
+   and a regular lat-lon grid for a single hour.
+   """
+   import metview as mv
+
+   surface_slice = ml_surface.sel(time=slice(datestring,datestring)).compute()
+   wind_slice = ml_wind.sel(time=slice(datestring,datestring)).compute()
+   moisture_slice = ml_moisture.sel(time=slice(datestring,datestring)).compute()
+
+   #fix the fieldsets
+   wind_fieldset = mv.dataset_to_fieldset(attribute_fix(wind_slice).squeeze())
+   surface_fieldset = mv.dataset_to_fieldset(attribute_fix(surface_slice).squeeze())
+   moist_fieldset = mv.dataset_to_fieldset(attribute_fix(moisture_slice).squeeze())
```

Comment:
Let's leave a warning here (and where appropriate) that explains that it will use a lot of local disk space. ecCodes will write gribs to disk instead of manipulating them in memory.

Reply:
Done.